parse/DoubleComplexValueRefParser.cpp

```diff
@@ -128,17 +131,17 @@ namespace parse {
         part_meter
             = ( tok.ShipPartMeter_
                 >   label(tok.Part_)    >   string_grammar
-                > label(tok.Name_) > string_grammar
-                //>   ship_part_meter_type_enum // probably Meter type
+                > label(tok.Meter_)
+                >   ship_part_meter_type_enum
```

Comment:
I think you could just use raw ( https://www.boost.org/doc/libs/1_45_0/libs/spirit/doc/html/spirit/qi/reference/directive/raw.html ) but I suppose this works as well...